sda-commons-server-jackson/README.md

```diff
@@ -29,6 +29,11 @@ and enables
 [`READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE`](https://static.javadoc.io/com.fasterxml.jackson.core/jackson-databind/2.9.7/com/fasterxml/jackson/databind/DeserializationFeature.html#READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE).
 The `FuzzyEnumModule` from Dropwizard is removed as it lacks support of newer Jackson features for enumerations.
 
+The framework already provides a predefined exception mapper for `InvalidTypeException` if the framework default 
```

Comment:
Maybe we move this information to the ErrorFormat section of the readme. Maybe we can rename ErrorFormat to Error Handling and add a hint, that exceptions from the code are catched by ExceptionHandlers and the body of the error is always transformed to a common ErrorFormat Structure.

Then we can add a section like
> The [`InvalidTypeIdExceptionMapper`](./src/main/java/org/sdase/commons/server/jackson/errors/InvalidTypeIdExceptionMapper.java) is only used if the ObjectMapper feature `FAIL_ON_INVALID_SUBTYPE` is is enabled  (See section `Customize the ObjectMapper`).

Reply:
I think your points already described in Ln 9-12. Maybe I move this to this section. Or what do you think?

Reply:
I moved the description now to the section above (after Ln 9-12)